requirements.txt

```diff
@@ -1,16 +1,17 @@
-# Module names are ordered alphabetically
-# Required packages are ordered alphabetically within each module
-
-# Global
-pytest
-
-# camera
-opencv-python
-numpy
-
-# comms
-libscrc
-pyserial
-
-# qr
-pyzbar
+colorama==0.4.6
+dronekit==2.9.2
+exceptiongroup==1.1.2
+future==0.18.3
+iniconfig==2.0.0
+libscrc==1.8.1
+lxml==4.9.3
+monotonic==1.6
+numpy==1.24.4
+opencv-python==4.8.0.74
+packaging==23.1
+pluggy==1.2.0
+pymavlink==2.4.39
+pyserial==3.5
+pytest==7.4.0
+pyzbar==0.1.9
+tomli==2.0.1
```

Comment:
We want to try to keep the `requirements.txt` organized, can you help us out with that? Also, I noticed there are some repeats in libraries here. Did you copy paste the output from pip list? I think a lot of these libraries might've been installed when you installed dronekit initially. 

I think you just need to add dronekit to the `requirements.txt`. To test that it is working correctly you can delete the `venv` folder and recreate the virtual environment and install requirements.txt to ensure all the dependencies are installed.